doc/Tutorials.xml

```diff
@@ -181,12 +181,108 @@ There are various examples of packages using &AutoDoc; for only
 this purpose, e.g. <Package>IO</Package> and <Package>orb</Package>.
 <P/>
 
-<!--
-TODO: Explain how to mix AutoDoc content with existing &GAPDoc; XML files.
--->
+<Subsection Label="Tut:IntegrateExisting:EverythingThere">
+  <Heading>Using &AutoDoc; on a complete &GAPDoc; manual</Heading>
+
+Suppose you already have a complete XML manual, with some main and title
+XML files and some documentation for operations distributed over
+all your <F>.g{,d,i}</F> files. Even then &AutoDoc; can do something for you.
```

Comment:
I am not sure people will understand this `.g{,d,i}` shorthand -- I'd spell it out as ".g, .gi and .gd files" or so